config/schema/acquia_lift.settings.yml

```diff
@@ -12,21 +12,12 @@ acquia_lift.settings:
         customer_site:
```

Comment:
I was confused with actual schemas for config entity storage. This is good as is.